shared/wallets/wallet/index.tsx

```diff
@@ -8,14 +8,6 @@ import Asset from '../asset/container'
 import Transaction from '../transaction/container'
 import Airdrop from '../airdrop/container'
 
-const stripePatternName = Styles.isMobile
```

Comment:
can't do this at the module level, needs to be done during render. also isDarkMode is a function